menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -180,6 +183,11 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
           hadoopFS.delete(hdfsConfPath, true)
       }
 
+      //For some reason these downloads were always cut half-way.. The blocking
+      //request is only made once per version (this step is skipped if jars are loaded already)
+      Await.ready(resFutureStd, Duration.Inf)
+      Await.ready(resFutureConf, Duration.Inf)
```

Comment:
Not a fan of infinite wait. I'd say this should be some configurable property like `download.duration.conformance.jars`.
Maybe also `logger.info("Downloading Standardization and Conformance jars, this may take some time.")`

Reply:
Agreed on the infinity waiting.
The `logger` is covered in the `downloadFile` function itself. But perhaps the wording could be changed. 🤔